evaluator.py

```diff
@@ -62,14 +63,17 @@ def __init__(self, config_path: str):
         with open(config_path, encoding="utf-8") as file:
             c = yaml.safe_load(file)
 
-        self.config = c
+        self.config = {
+            "IPERF_TCP": {item["id"]: item["threshold"] for item in c["IPERF_TCP"]},
+            "IPERF_UDP": {item["id"]: item["threshold"] for item in c["IPERF_UDP"]},
+        }
         self.test_results: List[TestResult] = []
 
-    def _eval_flow_test(self, run):
+    def _eval_flow_test(self, run) -> None:
         try:
             run = TftAggregateOutput(**run)
             data = IperfOutput(**run.flow_test)
-            md = TestMetadata(**data.tft_metadata)
+            md = from_dict(TestMetadata(**data.tft_metadata))
```

Comment:
this call lacks a `data` argument.

However, a followup commit refactors this again.